apiserver/facades/client/client/status.go

```diff
@@ -177,7 +177,7 @@ func (c *Client) FullStatus(args params.StatusParams) (params.FullStatus, error)
 	if context.status, err = context.model.LoadModelStatus(); err != nil {
 		return noStatus, errors.Annotate(err, "could not load model status values")
 	}
-	if context.applications, context.units, context.latestCharms, err =
+	if context.applications, context.units, context.latestCharms, context.endpointBindings, context.service, err =
```

Comment:
there's now enough of these that we should consider creating a new struct to hold the return values, eg applicationContext